source/dotnet/Library/AdaptiveCards.Rendering.Wpf/RenderedAdaptiveCard.cs

```diff
@@ -20,6 +20,10 @@ public RenderedAdaptiveCard(FrameworkElement frameworkElement, AdaptiveCard orig
 
         internal Dictionary<string, Func<object>> InputBindings { get; set; } = new Dictionary<string, Func<object>>();
 
+        /// <summary>
+        /// If any modifications were done to the rendered card they will be reported here
+        /// </summary>
+        public IList<AdaptiveViolation> Warnings { get; internal set; } = new List<AdaptiveViolation>();
```

Comment:
`AdaptiveViolation` seems like a scary name. Not sure it should be a "violation" if there was simply an unknown type encountered, for example?

Reply:
Agree, since it's just for warnings now I'll change the name